android-auto-app/build.gradle.kts

```diff
@@ -32,6 +32,11 @@ android {
     }
   }
 
+  // THIS IS REQUIRED or else it will not compile
```

Comment:
Inheriting from `MapboxCarMapObserver` does not compile without this compiler flag when an sdk is built with the `jvm-default=all` flag

Here is the compile error. It is nothing to do with `CarAnimationThreadController` except that it is inheriting from `MapboxCarMapObserver`. Also realize that it is the kotlin compiler that fails, not the `CarJavaInterfaceChecker`
```
e: /Users/kylemadsen/Development/mapbox/mapbox-maps-android/android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarAnimationThreadController.kt: (16, 7): Inheritance from an interface with '@JvmDefault' members is only allowed with -Xjvm-default option
```